src/main/app/src/constants/ActivitiesFilterBar.js

```diff
@@ -147,16 +158,21 @@ const ArticlesFilterBar = ({handleFilterChange, filterButtonClick, filterResetCl
 }
 
 /** Videos filter options. */
-const VideosFilterBar = ({handleFilterChange, filterButtonClick, filterResetClick, videoType}) => {
+const VideosFilterBar = ({handleFilterChange, filterButtonClick, filterResetClick, videoCategory, videoDuration}) => {
   return (
     <React.Fragment>
       <div className="field is-grouped">
         <div className="control is-expanded"><div className="select is-fullwidth">
-          <select name="VideoType" onChange={handleFilterChange} value={videoType}>
-            <option value="*">{"All"}</option>
+          <select name="VideoCategory" onChange={handleFilterChange} value={videoCategory}>
+            <option value="*">{"All [Video Category]"}</option>
             <option value="yoga">{"Yoga"}</option>
             <option value="workout">{"Workout"}</option>
             <option value="meditation">{"Meditation"}</option>
+          </select>
+        </div></div>
+        <div className="control is-expanded"><div className="select is-fullwidth">
+          <select name="VideoDuration" onChange={handleFilterChange} value={videoDuration}>
+            <option value="*">{"All [Video Duration]"}</option>
             <option value="short">{"Videos less than 15 minutes"}</option>
             <option value="medium">{"Videos greater than 15 minutes and less than 30 minutes"}</option>
             <option value="large">{"Videos greater than 30 minutes"}</option>
```

Comment:
It might be more readable without "Videos" and just having "less than 15 minutes", "between 15 and 30 minutes", and "greater than 30 minutes"?

Reply:
Done!